bee-protocol/src/milestone/tangle.rs

```diff
@@ -0,0 +1,165 @@
+use crate::milestone::MilestoneIndex as MsIndex;
```

Comment:
Can we create a `tangle` module for the tangle in protocol ? This module is only for the milestone definition.

Reply:
sure :+1: